Docker.DotNet/Endpoints/IMiscellaneousOperations.cs

```diff
@@ -21,8 +21,14 @@ public interface IMiscellaneousOperations
 
         Task<Stream> GetImageAsTarballAsync(string name, CancellationToken cancellationToken);
 
+        Task<Stream> GetMultipleImagesAsTarballAsync(string[] names, CancellationToken cancellationToken);
+
         Task LoadImageFromTarballAsync(Stream stream, CancellationToken cancellationToken);
 
+        Task<Stream> LoadImageFromTarballWithProgressAsync(Stream stream, CancellationToken cancellationToken);
```

Comment:
Can you explain the naming here?

What is the difference in the return stream from `LoadImageFromTarballWithProgressAsync` and `LoadImageFromTarball`?